docs/references/chrome-extension/overview.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Chrome Extension SDK
+description: Learn how to integrate Clerk into your Chrome Extension
+---
+
+# Clerk Chrome Extension SDK
+
+
+## Chrome Extension Key
+
+A Chrome Extension can be identified by its unique, in a similar way to how a website can be identified by its domain. You will need to explicitly configure your Chrome Extension's key or it will change often. If the key changes it can cause the Chrome Extension to fail.
```

Comment:
```suggestion
A Chrome Extension can be identified by its unique ID, in a similar way to how a website can be identified by its domain. You will need to explicitly configure your Chrome Extension's key or it will change often. If the key changes it can cause the Chrome Extension to fail.
```